project.json

```diff
@@ -75,6 +75,22 @@
           "target": "build"
         }
       ]
+    },
+    "run-history": {
+      "command": "npx dist/packages/cli history",
+      "dependsOn": [
+        {
+          "projects": [
+            "cli",
+            "plugin-eslint",
+            "plugin-coverage",
+            "plugin-js-packages",
+            "examples-plugins",
+            "react-todos-app"
+          ],
+          "target": "build"
+        }
+      ]
     }
```

Comment:
It's no longer need to configure individual Nx target to run CLI commands. Since I merged #574, there is now a single `code-pushup` target you can add any command to - e.g. `npx nx code-pushup -- history ...`.

Reply:
removed